### ISSUE TITLE
Odometer reading in kilometers, but units show miles

### ISSUE THREAD

**Issue by dbemowsk** [itchannel/fordpass-ha#270]
If this is the integration through HACS, the issue I found is that the FordPass app on my phone shows the odometer at 93,349 miles, but HA shows it at 150,232 miles.  93,349 miles converts to 150,230.65 kilometers.  Seems like an easy fix.
![image](https://github.com/itchannel/fordpass-ha/assets/2935181/2fde0eed-9c83-4431-b41f-b344e65f255e)

![image](https://github.com/itchannel/fordpass-ha/assets/2935181/0b1c18d4-8ca4-4630-b406-065c9e004f97)

**Comment by itchannel** [comment]
Then annoyance is for certain countries the API returns the mileage in KM and others in Miles. 

```
"odometer":{"value":2206.0,"status":"CURRENT","timestamp":"07-06-2023 02:59:11"
````

But gives no context on if the value is Miles or Kms. If you set the mileage option in settings to KM it will display in mileage but the unit at the end will be wrong. Out of interest can you tell me what country you are from as it may be country specific on what odometer type is returned?

**Comment by dbemowsk** [comment]
I am in the US.  I see now that it appears to be working.  On the day that I posted this, it didn't matter if I had the settings set to Mi or Km, the value shown was as shown in the original post.  I tried toggling between both a couple times.  I just tried it now and it shows the correct value.
![image](https://github.com/itchannel/fordpass-ha/assets/2935181/3eb6c7d9-d217-499a-9953-31046b579300)

**Comment by itchannel** [comment]
Fixed in 1.47

**Comment by dai80027** [comment]
I'm having the same issue in 1.48
